guides/trailblazer/2.0/01-operation-basics.md

```diff
@@ -10,14 +10,14 @@ redirect_from: "/guides/trailblazer/2.0/01-getting-started-with-operation.html"
   {{ "app/blog_post/operation/create.rb:failure:../trailblazer-guides:operation-01" | tsnippet : "impl" }}
 
   ~~~6
-  The *Operation* is the central concept of the Trailblazer architecture. It is a simple service object that embraces and orchestrates all the business logic necessary to accomplish a certain task, such as creating a blog post, or updating a user.
+  The *Operation* is the central concept of the Trailblazer architecture. It's a simple service object that encapsulates and executes all the business logic necessary to accomplish a certain task, such as creating a blog post, or updating a user.
```

Comment:
I'm not sure what the right word is, but I've never really heard 'embrace' used before to talk about code, so it sounded weird here.

Is it really true though that the business logic isn't happening inside the operation? If I define a step as an instance method (`step :my_method`), wouldn't this part of the business logic now be within the operation? (Within `my_method`)